src/shared/telemetry/telemetryUtils.ts

```diff
@@ -42,15 +57,16 @@ export function registerCommand<T>({
                 throw e
             } finally {
                 const endTime = new Date()
-                const datum = result && result.datum ? result.datum : defaultMetricDatum(command)
+                const datum = result && result.datum ?
```

Comment:
Please consider unraveling this statement so it does not contain nested `?:` operators so that it is easier to read/maintain at a glance.

Reply:
✔️